src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Route, Routes } from 'react-router-dom';
+import AddBook from './components/addbook';
+import Categories from './components/categories';
+import Navbar from './components/navbar';
+import './App.css';
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<AddBook />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+    </>
```

Comment:
- [ ] Please, consider that it is required the App component to be as follows:
* Has 2 `<Route>`s, one for the BookList component and the other for the Categories component

- You have only the `AddBook` component instead of the `Books` component, so it means you have to replace it 👍  So you have to fix the structure of the components as required in this project. I now, you can do it easily 💯 👍